library/Rules/MinimumAge.php

```diff
@@ -20,16 +21,16 @@ class MinimumAge extends AbstractRule
 
     public function __construct($age, $format = null)
     {
+        if (!filter_var($age, FILTER_VALIDATE_INT)) {
+            throw new ComponentException('The age must be a numeric value.');
+        }
+
         $this->age = $age;
         $this->format = $format;
     }
 
     public function validate($input)
     {
-        if (!is_int($this->age)) {
```

Comment:
Yes, according to our [Release Cycle and Support](https://github.com/Respect/Validation/wiki/Release-Cycle-and-Support) we're still supporting 0.8, 0.9 and 1.0 as  well, of course.